scripts/ldev2pcs

```diff
@@ -102,10 +126,34 @@ def configure_location(resource, nodes, fixed_score_string=None):
             score -= 10
 
 def configure():
-    cmd = 'pcs cluster setup --force --local'
-    cmd += ' --name '+cmdline_args.cluster_name
-    cmd += ' '+cmdline_args.mgmt_node
-    run(cmd)
+    # the syntax for making the corosync.conf
+    # file depends on the pacemaker version
+    pacemaker_major_number = get_pacemaker_major_number()
+    if pacemaker_major_number == 1:
+        cmd = 'pcs cluster setup --force --local'
+        cmd += ' --name '+cmdline_args.cluster_name
+        cmd += ' '+cmdline_args.mgmt_node
+    if pacemaker_major_number == 2:
```

Comment:
Should this be "elif", so the "else" clause is entered only for pacemaker !=1 and !=2?

Reply:
Thanks, I moved the ValueError from before to after the cmd construction and forgot to change that.